cypress/integration/cypress_actions.js

```diff
@@ -16,3 +50,72 @@
 // 8. Check search by all column values
 
 // https://demoqa.com/webtables
+
+describe('Advanced level', () => {
+    beforeEach(() => {
+        cy.visit('https://demoqa.com/webtables');
+    });
+
+    it('Pagination', () => {
+        cy.get('.-pageInfo').contains('1');
+    });
+
+    it('Rows count selection', () => {
+        cy.get('select').select('10 rows');
+    });
+
+    it('Add new worker', () => {
+        cy.get('#addNewRecordButton').click();
+        cy.get('#firstName').type('Lola');
+        cy.get('#lastName').type('Kiska');
+        cy.get('#userEmail').type('dodik@shmodik.com')
+        cy.get('#age').type('31');
+        cy.get('#salary').type('160000');
+        cy.get('#department').type('QA')
+        cy.get('#submit').click();
+
+    });
+
+    it('Delete worker', () => {
+        cy.get('#delete-record-1 > svg > path').click();
+    })
+
+    it('Delete all worker', () => {
+        cy.get('#delete-record-1 > svg > path').click();
+        cy.get('#delete-record-2 > svg > path').click();
+        cy.get('#delete-record-3 > svg > path').click();
+    })
+
+    it('Find worker in search field and edit it', () => {
+        cy.get('#searchBox').type('Kierra{enter}');
+        cy.get('#edit-record-3 > svg > path').click();
+        cy.get('#lastName').type('{selectall}').type('Zhuzhuka');
+        cy.get('#submit').click();
+        
+    });
+
+    it('Validate data in worker row after creating worker', () => {
+        cy.get('#addNewRecordButton').click();
+        cy.get('#firstName').type('Lola');
+        cy.get('#lastName').type('Kiska');
+        cy.get('#userEmail').type('dodik@shmodik.com')
+        cy.get('#age').type('31');
+        cy.get('#salary').type('160000');
+        cy.get('#department').type('QA')
+        cy.get('#submit').click();
+        cy.get(':nth-child(4) > .rt-tr > :nth-child(1)').contains('Lola');
+    });
+
```

Comment:
try to find some real data in the search bar. your queries find nothing for now. this is useless